starter-code/pizza.js

```diff
@@ -1 +1,58 @@
 // Write your Pizza Builder JavaScript in this file.
+
+
+var $pizza = $('#pizza');
+var $price = $('.panel li');
+
+/***** Pepperonni ****/
+
+var $peppeButton = $('.btn-pepperonni');
+var $allPeppe = $('.pep');
+
+$peppeButton.on('click', function(){
+    $allPeppe.toggle();
+    $peppeButton.toggleClass('active');
+});
+
+
+/***** Mushroom ****/
+
+var $mushButton = $('.btn-mushrooms');
+var $allMush = $('.mushroom');
+
+$mushButton.on('click', function () {
+    $allMush.toggle();
+    $mushButton.toggleClass('active');
+})
+
+/***** Green Peppers ****/
+
+var $greenButton = $('.btn-green-peppers');
+var $allGreen = $('.green-pepper');
+
+$greenButton.on('click', function () {
+    $allGreen.toggle();
```

Comment:
Do you see how you repeat the same code again and again. Maybe you can abstract that into a function.